index.html

```diff
@@ -2168,10 +2171,7 @@ <h2>Definition of Roles</h2>
 					<tr>
 						<th class="role-related-head" scope="row">Related Concepts:</th>
 						<td class="role-related">
-							<ul>
-								<li><a href="https://www.w3.org/TR/html5/grouping-content.html#the-dd-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>dd</code></a></li>
-								<li><a href="https://www.w3.org/TR/html5/text-level-semantics.html#the-dfn-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>dfn</code></a></li>
-							</ul>
+              <a href="https://www.w3.org/TR/html5/text-level-semantics.html#the-dfn-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>dfn</code></a>
```

Comment:
see: https://github.com/w3c/aria/pull/951#issuecomment-486914805

Reply:
@scottaohara 
I think you have a good point here, the current ARIA spec is not aligned with HTML5 spec for the ARIA `term` and `definition` roles, see [HTML dfn element](https://www.w3.org/TR/html5/textlevel-semantics.html#the-dfn-element)

The following example show how the `definition` and `term` roles should be used to line up with the HTML5 definition of `dfn` element:

```
<div role="definition">
   An <span role="term">electron</span> is a stable subatomic particle with a charge of negative electricity, found in all atoms and acting as the primary carrier of electricity in solids.
</div>
```

Reply:
Interesting.  per the current prose for these roles, I was thinking something like the following was necessary:

```
<p>
  <span role="term" id="e">Electron</span> <span role="definition" aria-labelledby="e">is a stable subatomic particle with a charge of negative electricity, found in all atoms and acting as the primary carrier of electricity in solids.</span>
</p>
```

I like your example. it makes sense and indicates to me that the `aria-labelledby` woudln't be necessary ( which is good imo since developers hate adding IDs to things :) )

Reply:
See offshoot PR #971